internal/generator/generator.go

```diff
@@ -66,9 +68,15 @@ func (g *Generator) Generate(examples ...*linker.LinkedExample) []*Suite {
 		var depsToSetup = Dependencies([]Dependency{Dependency(g.conf.BasePkg)})
 		depsToSetup = append(depsToSetup, normalizeDeps(moduleName, e.ParentDependencies())...)
 
+		location := filepath.Join(g.conf.OutputDir, strings.ToLower(e.Name))
+		if g.conf.Bash {
+			location = filepath.Join(location, "suite.sh")
```

Comment:
It would probably be better to use `suite.gen.sh` name, to indicate that the file is generated.